app/editor/src/components/form/select/Select.tsx

```diff
@@ -51,6 +52,8 @@ export interface ISelectProps<OptionType> extends SelectProps {
   ref?: Ref<ReactSelect<OptionType, boolean, GroupBase<OptionType>>>;
 }
 
+type TooltipRef = { tooltipRef: null } | null;
```

Comment:
Is there a better place to put this? Maybe utils? Wasn't sure where; I believe we will not need to use a ref outside of the select component.